docs/docs/guides/CODE_SCANNING.mdx

```diff
@@ -37,7 +37,7 @@ On Android, the [MLKit Vision Barcode Scanning](https://developers.google.com/ml
       ...
       <meta-data
           android:name="com.google.mlkit.vision.DEPENDENCIES"
-          android:value="barcode">
+          android:value="barcode" />
```

Comment:
Are you 100% sure this is correct? In the Google MLKit documentation they also don't have a closing tag:

![image](https://github.com/mrousavy/react-native-vision-camera/assets/15199031/ca5b2b67-bf81-472f-b341-0a569a53236b)

https://developers.google.com/ml-kit/vision/barcode-scanning/android

Reply:
As described in #1922, it does not compile for me at least if it isn't a syntactically correct XML closing tag, with the error `com.android.manifmerger.ManifestMerger2$MergeFailureException: Error parsing /[...]/AndroidManifest.xml` indicating it's a parsing error.

I'm not user how to address this otherwise.

Reply:
Cool - @NTKrieger can you revert the other change then we can merge this XML fix!